### ISSUE TITLE
XSD allows multiple comment tags while only the last is taken into account

### ISSUE THREAD

**Issue by Vampire** [liquibase/liquibase#1740]
## Environment
**Liquibase Version**: 4.3.1

**Liquibase Integration & Version**: n/a

**Liquibase Extension(s) & Version**: n/a 

**Database Vendor & Version**: n/a

**Operating System Type & Version**: n/a

## Description
The XSD seems to intend to enforce only one comment tag in a change set, but it does not do so.
Change logs with multiple comment tags per change set validate perfectly fine against the 4.3 or 3.6 XSD, but all comments except for the last are simply ignored.
The schema should prevent that.

**Comment by molivasdat** [comment]
Thanks @Vampire for writing up this issue.  We will add it to the list of changes that we are processing.

**Comment by FBurguer** [comment]
Looks like this isse got fixed for the latest XSD version `4.15`.
Changeset:
```
        <changeSet  id="1"  author="nvoxland">
                <comment>A sample change log1</comment>  
                <comment>A sample change log2</comment>  
                <comment>A sample change log3</comment> 
          </changeSet>
```
updateSQL output:
```
-- Changeset changelogs29/changelog.xml::1::fburguer
-- A sample change log1
-- A sample change log2
-- A sample change log3
```
